File: PrimeNums.py
#Print the prime numbers between 100 and 200

def print_primes():
    result = []
    for i in range(100, 201):
        is_prime = True
        for j in range(2, i):
            if i % j == 0:
                is_prime = False
        if is_prime == False:
            continue
        else:
            result.append(i)
    return result

print(print_primes())

File: PrimeNums.js
//Print the prime numbers between 100 and 200

let primeNums = () => {
    var isPrime;
    for (let i=100; i<201; i++) {
        isPrime = true;
        for (let j=2; j<i; j++) {
            if (i % j === 0) {
                isPrime = false;
            }
        }
        if (isPrime) {
            console.log(i);
        }
    }
}

primeNums();